src/OpenTelemetry.AutoInstrumentation.Native/calltarget_tokens.h

```diff
@@ -31,8 +31,8 @@ class CallTargetTokens
 {
 private:
     ModuleMetadata* module_metadata_ptr = nullptr;
-    const bool enable_by_ref_instrumentation = false;
-    const bool enable_calltarget_state_by_ref = false;
+    const bool enable_by_ref_instrumentation = true;
```

Comment:
Could you please add comment that we decided to keep it to make upstream sync easier?

Reply:
Added in bc10d076daf14543982643b5269c9566b811d6ee

Reply:
Added in bc10d076daf14543982643b5269c9566b811d6ee